requirements.txt

```diff
@@ -3,3 +3,6 @@ kubernetes>=27.2.0 # introduction of suspend in jobspec
 flask>=2.0.0
 natsort>=8.0.0
 Flask-BasicAuth>=0.2.0
+MarkupSafe>=2.1.5
+apache-libcloud>=3.8.0
+requests>=2.31.0
```

Comment:
requests is not a direct dependency, why do you need to add this? (and if you need, then it would make sense to document it in the requirements file)